controllers/home.js

```diff
@@ -13,6 +13,15 @@ async function home(req, res) {
     csrfToken: req.csrfToken(),
   };
 
+  const coinFlipNumber = Math.random() * 100;
+  const experimentBranch = getExperimentBranch(req, coinFlipNumber);
+
+  const isUserInExperiment = isExperiment(experimentBranch);
+  let experimentBranchB = false;
```

Comment:
I think this could also just be `const experimentBranchB = (experimentBranch === "vb" && isUserInExperiment)`

Reply:
Revised!